### PR TITLE
add pointing correction in preproc module

### PR DESCRIPTION
**Pull request by susannaaz**

### CODE REVIEW

**Review by chervias — APPROVED** [review]
I just tested and it seems to be working fine.

**Review by msilvafe — CHANGES_REQUESTED** [review]
Please address Carlos + my comments then re-request review.

File: sotodlib/preprocess/processes.py [hunk 1]
```diff
@@ -1355,6 +1355,25 @@ def save(self, proc_aman, split_flg_aman):
...
+          calc: True
+          
+    .. autofunction:: sotodlib.hwp.hwp_angle_model.apply_hwp_angle_model
+    """
+    name = "pointing_model"
+    
+    def calc_and_save(self, aman, proc_aman):
+        from sotodlib.coords import pointing_model
+        if self.calc_cfgs:
+            pointing_model.apply_pointing_model(aman)
```
[the remark this answers is not in the document]
Comment (chervias) [reply sotodlib/preprocess/processes.py:1375]: ah ok. I didn't check that it worked, haha. I just checked that it didn't break the mapmaker.

File: sotodlib/preprocess/processes.py [hunk 2]
```diff
@@ -1355,6 +1355,25 @@ def save(self, proc_aman, split_flg_aman):
...
+    """Apply pointing model to the TOD.
+
+    Saves results in proc_aman under the "pointing" field. 
+
+     Example config block::
+
+        - name : "pointing_model"
+          calc: True
+          
+    .. autofunction:: sotodlib.hwp.hwp_angle_model.apply_hwp_angle_model
```
[the remark this answers is not in the document]
Comment (susannaaz) [reply sotodlib/preprocess/processes.py]: Yep, sorted

File: sotodlib/preprocess/processes.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (susannaaz) [reply sotodlib/preprocess/processes.py]: Done

**Review by msilvafe — APPROVED** [review]
Looks good!

**Pull request merged**